WEB-4596 Removed unwanted constants for affiliates:

pvkadam opened the pull request:
Added another intro files for affiliates removing the 2 unwanted constants.

pvkadam committed:
WEB-4596 Removed unwanted constants for affiliates:
Added another intro files for affiliates removing the 2 unwanted constants.

gabe-kent commented on source/api_documentation/transactions_api/_intro_for_affiliates.rst:
```diff
@@ -0,0 +1,68 @@
+Typical usage on the polling interval is to repeatedly call the API until no rows are returned, meaning you have downloaded all transactions.
+Please note, the "to" and "from" date range parameters are both necessary, providing only one or the other will not filter the results.
+
+We provide a helpful constant that can be used in the include_columns and exclude_columns options:
```
All of the information above this line is common between advertiser and network documentation.

If a change needs to be made to the above then it will require making the same change in 2 places: this file and the original _intro.rst file. That is unideal. We want to DRY up the above information so that we only have to change 1 file if the information needs to be changed/updated.

I believe my suggestion that we discussed would result in a much DRYer solution.

1. Keep the original _intro.rst file
2. Move the section from this line down into it's own file: `_affiliate_intro_constants.rst`
3. Move the same section from the _intro.rst file to a new file: `_intro_constants.rst`
4. Ensure the advertiser and network documentation specify `.. include:: _intro_constants.rst` after `.. include:: _intro.rst`
5. Ensure the `affiliate_user.rst` file specifies `.. include:: _affiliate_intro_constants.rst` after `.. include:: _intro.rst`

DRY == **D**o not **R**epeat **Y**ourself

gabe-kent reviewed (changes requested):
Nice work, though I think we could make this DRYer!

pvkadam committed:
WEB-4596 Removed unwanted constants for affiliates:
Made code more DRYer by combining 2 intro files with same content except for Constants section.

echistyakov commented on source/api_documentation/transactions_api/advertiser_user.rst:
```diff
@@ -26,6 +26,8 @@ The API follows REST conventions. Perform an HTTPS GET to the URL with the forma
 
 .. include:: _intro.rst
+.. include:: _intro_constants.rst
+
```
Looks like an extra new line here (not sure if this affects the resulting formatting)

echistyakov commented on source/api_documentation/transactions_api/network_user.rst:
```diff
@@ -33,6 +33,8 @@ The API follows REST conventions. Perform an HTTPS GET to the URL with the forma
 
 .. include:: _intro.rst
+.. include:: _intro_constants.rst
+
```
Likewise

echistyakov reviewed (approved):
Looks good 👍  Minor comment about an extra new line.

pvkadam committed:
WEB-4596 Removed unwanted constants for affiliates:
Removed extra white spaces.
